src/styles/ThemeStylesProvider.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, {useMemo} from 'react';
+import PropTypes from 'prop-types';
+import useTheme from './themes/useTheme';
+import StylesContext from './ThemeStylesContext';
+import defaultStyles from './styles';
+
+const propTypes = {
+    children: PropTypes.node.isRequired,
+};
+
+function ThemeStylesProvider(props) {
+    const theme = useTheme();
+
+    const appContentStyle = useMemo(
+        () => ({
+            ...defaultStyles.appContent,
+            backgroundColor: theme.appBG.value,
```

Comment:
```
            backgroundColor: theme.appBG.value,
```
Can you explain this line

Reply:
Sorry, this is a leftover from the REA implementation before.

This is where the dynamic style gets initialized and memoized